.travis.yml

```diff
@@ -21,6 +21,7 @@ before_install:
       AUTO_DEPLOY=true;
     else
       AUTO_DEPLOY=false;
+      AWS_BUCKET="repositoryfinder.stage.datacite.org";
```

Comment:
This line is not needed, as the code will not be deployed.

Reply:
Yes but I couldn't get it to work locally without defining AWS_BUCKET.